package.json

```diff
@@ -15,9 +15,7 @@
     "start": "node dist/index.js",
     "migrate": "tsx src/cli.ts",
     "test": "node --experimental-vm-modules --disable-warning=ExperimentalWarning node_modules/jest/bin/jest.js",
-    "test:node20": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
This was only a placeholder till CI got put in; it's now inline in the pipeline (same below)